ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Wallets.hs

```diff
@@ -192,10 +192,11 @@ deleteHdWallet pw rootId = do
             Keystore.delete (WalletIdHdSeq rootId) (pw ^. walletKeystore)
             return $ Right ()
 
-removeKeysFromKeyStore :: PassiveWallet
+removeKeysFromKeystore
+ :: PassiveWallet
```

Comment:
Now indentation looks very weird.

Reply:
Why was it marked as resolved? It's 1-space indentation here. It's not allowed.

Reply:
Hm, strange, I was pretty sure I was looking at the latest version last time I saw it. Ok then.